Sources/App/Views/PackageController/PackageShow+Model.swift

```diff
@@ -350,6 +350,15 @@ extension PackageShow.Model {
                 .class("matrix"),
                 .forEach(rows) { compatibilityListItem(label: $0.label, cells: $0.results.cells) }
             ),
+            .p(
+                .id(
+                    // TODO: remove check when we go live
+                    ((try? Environment.detect()) ?? .development) == .production
+                        ? "ignored"
+                        : "swift_version_badge"),
+                .text(badgeMarkdown(for: .swiftVersions)),
+                .attribute(named: "hidden")
+            ),
```

Comment:
I ended up sticking it in `p` tag, because the markdown wouldn't go in the `href` (got prefixed with the package url - I guess it has to be a url or something).

This makes the button show up in a new line. But even as an `<a href=...` the button was still misplaced - it came after the "Full build logs" link, because it's attached like so: `element.parentNode.appendChild(copyButtonElement)`

Probably an easy fix, and I may dig around a bit but I wanted to push what we've got for now.